cmake/modules/AddSwift.cmake

```diff
@@ -159,7 +159,7 @@ function(_add_variant_link_flags
       result)
 
   if("${sdk}" STREQUAL "LINUX")
-    list(APPEND result "-lpthread" "-ldl")
+    list(APPEND result "-lpthread" "-ldl" "-Wl,-Bsymbolic")
```

Comment:
`-Bsymbolic` seems to be a big hammer (and it might be just a workaround for what is an LLVM or a linker bug).  Could you scope it as narrowly as possible, just to armv6?